src/test/java/net/kaczmarzyk/spring/data/jpa/Customer.java

```diff
@@ -83,6 +83,7 @@ public class Customer {
     private OffsetDateTime dateOfNextSpecialOffer;
     private ZonedDateTime dateOfNextSpecialOfferZoned;
 
+    @Column(columnDefinition = "uuid")
```

Comment:
Added to solve problem with UUID tests
https://hibernate.atlassian.net/browse/HHH-11490?focusedCommentId=106793